docs/core/introduction.md

```diff
@@ -1,238 +1,84 @@
 
-Project file:
+Value types and stack-allocated memory blocks offer more direct, low-level control over data and native platform interop, in contrast to .NET’s GC-managed types. Most of the primitive types in .NET, like integer types, are value types, and users can define their own types with similar semantics. Value types are fully supported through .NET’s generics system, meaning that generic types like `List<T>` can provide flat, no-overhead memory representations of value type collections.
 
-```xml
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net8.0</TargetFramework>
-  </PropertyGroup>
-</Project>
-```
+[Reflection](/dotnet/csharp/advanced-topics/reflection-and-attributes/) is a “programs as data” paradigm, allowing one part of a program to dynamically query and/or invoke another, in terms of assemblies, types and members. It is particularly useful for late-bound programming models and tools.
 
-Source code:
+Exceptions are the primary error handling model in .NET. Exceptions have the benefit that error information does not need to be represented in method signatures or handled by every method. Proper exception handling is essential for application reliability. Expected exceptions can be intentionally handled in user code, otherwise an app will crash. A crashed app is more reliable and diagnosable than an app with undefined behavior.
 
-```csharp
-Console.WriteLine("Hello, World!");
-```
+App stacks, like ASP.NET Core and Windows Forms, build on and take advantage of low-level libraries, language, and runtime. The app stacks define the way that apps are constructed and their lifecycle of execution.
 
-The app can be built and run with the [.NET CLI](tools/index.md):
+The SDK and other tools enable a modern developer experience, both on a developer desktop and for Continuous Integration (CI). This includes being able to build, analyze, and test code. .NET projects can often be built by a single `dotnet build` command, which orchestrates restoring NuGet packages and building dependencies.
 
-```dotnetcli
-dotnet run
-Hello, World!
-```
-
-## Binary distributions
-
-* [.NET SDK](sdk.md): Set of tools, libraries, and runtimes for development, building, and testing apps.
-* [.NET Runtimes](https://dotnet.microsoft.com/download/dotnet): Set of runtimes and libraries, for running apps.
-
-You can download .NET from:
-
-* [The Microsoft download site](https://dotnet.microsoft.com/download).
-* [Containers](https://mcr.microsoft.com/catalog?search=dotnet).
-* [Linux package managers](install/linux.md).
+NuGet is the package manager for .NET. It contains hundreds of thousands of packages that implement functionality for many scenarios. The majority of apps rely on NuGet packages for some functionality. The [NuGet Gallery](https://nuget.org/) is maintained by Microsoft.
```

Comment:
```suggestion
NuGet is the package manager for .NET. It contains hundreds of thousands of packages that implement functionality for many scenarios. A majority of apps rely on NuGet packages for some functionality. The [NuGet Gallery](https://nuget.org/) is maintained by Microsoft.
```

Reply:
I'm confused by this change. I have never seen "A majority" in any written text. Is that a styleguide thing?

Reply:
I think "a" sounds better here, but either is probably fine. My reasoning is that it's not about a well-known group of apps, where "the" would be more appropriate (e.g. the majority oppresses the minority). Related: https://ell.stackexchange.com/questions/38244/what-is-the-difference-in-meaning-between-a-majority-of-and-the-majority-of

Reply:
Interesting. Thanks for the link. I committed the suggestion.